.github/workflows/test.yml

```
@@ -65,3 +65,46 @@ jobs:
         uses: actions/checkout@v2
       - name: Check Go module consistency
         run: ./tools/check_go_mod.sh
+
+  build:
+    name: Build binary
+    runs-on: 'ubuntu-latest'
```

```suggestion
    runs-on: ubuntu-latest
```
FYI there is no need for `'` nor `"` for YAML strings

---

Done